.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "tests/data/cpp/DeepSpeech"]
+	path = tests/data/cpp/DeepSpeech
+	url = https://github.com/mozilla/DeepSpeech.git
+[submodule "tests/data/rust/grcov"]
+	path = tests/data/rust/grcov
+	url = https://github.com/mozilla/grcov.git
```

Comment:
I think it'd be preferable to avoid submodules, and simply clone the repos at the time we have to run the tests. This way we don't force everyone to clone them even if they don't need to run the intergration tests, and the repository stays smaller.

Reply:
I will try to replace submodules with git clone commands at the beginning of my tests. In that way repositories are downloaded as first thing while running the tests (if not already present locally). Together with the repositories, I will also try to clone a specific version of RCA during my tests, so I can use it to produce the json files.

Thanks a lot for the suggestions!

Reply:
> I will also try to clone a specific version of RCA during my tests, so I can use it to produce the json files.

We can also store them in a separate repo, it might actually be better:
1) We avoid running RCA twice (before and after)
2) We have a history of how the values change over time with the evolution of RCA

Reply:
You're right, I can try that!
Where should I create the repository containing all the json files?

Reply:
You can create it in your personal GitHub account for now

Reply:
Michele gave me the permission to create the repository at https://github.com/SoftengPoliTo, so I will try to create it there.
Thanks for the support!